modules/services/lorri.nix

```
@@ -60,6 +90,30 @@ in {
 
         Install = { WantedBy = [ "sockets.target" ]; };
       };
+
+      services.lorri-notify =
+        mkIf (pkgs.stdenv.hostPlatform.isLinux && cfg.notify) {
```

No need to check for Linux platform
```suggestion
        mkIf cfg.notify {
```